frontend/javascripts/oxalis/model/accessors/tool_accessor.ts

```diff
@@ -112,13 +112,48 @@ function _getDisabledInfoWhenVolumeIsDisabled(
   };
 }
 
+function _getDisabledInfoForProofreadTool(
+  hasSkeleton: boolean,
+  agglomerateState: AgglomerateState,
+  isProofReadingToolAllowed: boolean,
+  isUneditableMappingPinned: boolean,
+  activeOrganization: APIOrganization | null,
+  activeUser: APIUser | null | undefined,
+) {
+  const isDisabled =
+    !hasSkeleton ||
+    !agglomerateState.value ||
+    !isProofReadingToolAllowed ||
+    isUneditableMappingPinned;
+  let explanation = "Proofreading actions are not allowed with a pinned and modified mapping.";
+  if (!isUneditableMappingPinned) {
+    if (!agglomerateState.value) {
+      explanation = agglomerateState.reason;
+    } else if (!isProofReadingToolAllowed) {
+      explanation = getFeatureNotAvailableInPlanMessage(
+        PricingPlanEnum.Power,
+        activeOrganization,
+        activeUser,
+      );
+    } else {
+      explanation = disabledSkeletonExplanation;
+    }
+  }
```

Comment:
> @MichaelBuessemeyer could you disable the whole proofreading tool in the frontend once the mapping is pinned (unless it is already a proofreading annotation i.e. mappingIsEditable==true)? Proofreading actions will fail after the first brush stoke as well.


I hope this covers all cases you mentioned @fm3? In short: The code checks whether there is a pinned but not an editable mapping. If that's the case, the proofreading tool is disabled with the message `"Proofreading actions are not allowed with a pinned and modified mapping."`

Reply:
Sounds good :) How about something like "Proofreading actions are not supported after modifying the segmentation."? Since it’s the brushing that’s the problem, not the having a mapping.